pkg/ingester/client/custom.go

```diff
@@ -59,3 +59,7 @@ func ChunkFromMeta(meta chunks.Meta) (Chunk, error) {
 
 	return ch, nil
 }
+
+func DefaultMetricsMetadataRequest() *MetricsMetadataRequest {
```

Comment:
Can you add a short comment explaining what this default value does? i.e. that it doesn't impose any limits